### DISCUSSION TITLE
Change the name of  the Add button in  child grid.

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#6013] [Help] [answered]
![image](https://user-images.githubusercontent.com/44040606/136105177-7e3fb65c-8610-4d6f-b775-9be1e9cb93f0.png)

How to change the button label "Add Budget Items"  to "Add Items" or something else?

**Comment by wezmag** [comment, marked as answer]
In your editor class, add below code

```
protected getAddButtonCaption() {
        return "Add Items";
}
```

**Reply by ghost** [reply]
Working.  Thanks!
